collectd/files/collectd_logrotate.conf

```diff
@@ -0,0 +1,11 @@
+/var/log/{{ service_name }}.log {
+  rotate 5
+  daily
+  nocreate
+  missingok
+  compress
+  delaycompress
+  minsize 20M
+  maxsize 50M
+  su root syslog
```

Comment:
I was testing using `logrotate /etc/logrotate.d/collectd` and the inheritance mechanism does not work in that case.

I removed the `su` option as you suggested.